keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all of the operation values; encrypt, decrypt, sign, verify, wrapKey and unwrapKey. For more information, see JsonWebKeyOperation.",
```

Comment:
@johanste - would you please advise on options. I've also noted the dev, @pomortaz, for her thoughts. I think its the convention that I'm asking about here. What should the convention for this case be? Thanks - Bruce